Formula/minetest.rb

```diff
@@ -29,11 +29,16 @@ class Minetest < Formula
   depends_on "cmake" => :build
   depends_on "freetype"
   depends_on "gettext"
+  depends_on "hiredis"
   depends_on "irrlicht"
   depends_on "jpeg"
+  depends_on "leveldb"
   depends_on "libogg"
   depends_on "libvorbis"
   depends_on "luajit"
+  depends_on "postgresql"
+  depends_on "spatialindex"
```

Comment:
Can you explain a bit behind the extra dependencies. Are they optional? If so, what benefit do each bring?